modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -44,6 +44,9 @@ AbaqusUMATStress::validParams()
   params.addParam<MooseEnum>("decomposition_method",
                              ComputeFiniteStrain::decompositionType(),
                              "Method to calculate the strain kinematics.");
+  params.addCoupledVar("displacements", 0.0, "displacement variables");
+  params.addParam<bool>(
+      "displace_qps", false, "Send displaced quadrature point coordinates to UMAT.");
```

Comment:
@jessecarterMOOSE Per your comment in the discussion, it seems that passing the displaced coordinates is the typical contract with the user-developed interface: Should this parameter be `true` by default?

Reply:
True, but then you'd have to have explicitly turn that off when running small strain models, right? Does it make sense to even have this option in a small strain condition? The more I think about it, it should probably follow the strain model (though I'm not sure how to handle that).

Reply:
This is already The `use_displaced_mesh` parameter (that every Material has) already does the correct thing.  You can default it to true (by overriding it's default value in `validParams`) but it will only give you the displaced mesh as `_mesh` _if_ the displaced mesh exists.

But maybe I don't understand the issue!

Reply:
@friedmud see our discussion below.

Reply:
@friedmud All I'm really after is getting the coordinates of the QP on the displaced mesh.